src/status_im2/contexts/onboarding/profiles/view.cljs

```diff
@@ -197,18 +200,18 @@
       [quo/input
        {:type              :password
         :blur?             true
-        :override-theme    :dark
+        :disabled?         processing
         :placeholder       (i18n/label :t/type-your-password)
         :auto-focus        true
-        :error?            (when (not-empty error) error)
+        :error?            (seq error)
         :label             (i18n/label :t/profile-password)
-        :secure-text-entry true
```

Comment:
Hi @ulisesmac, why we removing this?

Reply:
im more wondering why we had this before

Reply:
oh prop already in component, never mind. Thank you for fixing issue

Reply:
> im more wondering why we had this before

https://developer.apple.com/documentation/uikit/uitextinputtraits/1624427-securetextentry

Reply:
> oh prop already in component, never mind. Thank you for fixing issue

yep! that :smile:

Reply:
nice